routes/database.rb

```diff
@@ -35,6 +35,7 @@ def self.registered(app)
           MyLog.log.fatal 'routes/database: Database Resource not found [edit]'
           halt 404, 'database resource not found'
         end
+        @project = Project.filter(id: @db.project_id).all.first
         @projects = Project.all
```

Comment:
it looks  like `@projects` being used to populate the project pull-down list.. it feels a bit redundant to query for one project and also query for all of them. I think you can just get the specific project out of `@projects` - that's a Sequel dataset, so you can filter it without doing a new query. So get Projects.all, then get this specific project out of that dataset like `@projects.where(:id => @db.project_id)`